invenio/modules/pages/admin.py

```diff
@@ -33,6 +33,8 @@
 
 def template_exists(form, field):
     """Form validation: check that selected template exists."""
+    if not field.data:
```

Comment:
what about adding 'required' validator to https://github.com/inveniosoftware/invenio/pull/3337/files#diff-b83bf3bae0fb93452518c475004ffe97R66 instead?